src/libraries/System.Text.Json/tests/Common/CollectionTests/CollectionTests.AsyncEnumerable.cs

```diff
@@ -18,18 +18,23 @@ public abstract partial class CollectionTests
         [MemberData(nameof(GetAsyncEnumerableSources))]
         public async Task WriteRootLevelAsyncEnumerable<TElement>(IEnumerable<TElement> source, int delayInterval, int bufferSize)
         {
+            if (StreamingSerializer is null)
+            {
+                return;
+            }
+
             JsonSerializerOptions options = new JsonSerializerOptions
             {
                 DefaultBufferSize = bufferSize
             };
 
-            string expectedJson = await JsonSerializerWrapperForString.SerializeWrapper(source);
+            string expectedJson = JsonSerializer.Serialize(source);
```

Comment:
Why were these changed from wrappers to direct calls?

Reply:
I assume no desire to wrap since these are just for obtaining expected JSON for the actual test,

Reply:
Yep, that's the underlying motivation here.